### ISSUE TITLE
Allow log file of job to be viewed in full-screen.

### ISSUE THREAD

**Issue by ryanstull** [azkaban/azkaban#2230]
I would be nice if you could click on a button to see the raw log.  Having to view a large log file through the smaller window azkaban provides by default is sub-optimal.

**Comment by burgerkingeater** [comment]
I agree, or allow users to download the complete log. Are you interested in contributing a fix?

**Comment by ryanstull** [comment]
Unfortunately I don't really have the time right now.
